spec/native-compile-cache-spec.coffee

```diff
@@ -68,12 +68,12 @@ describe "NativeCompileCache", ->
 
   describe "when a previously required and cached file changes", ->
     beforeEach ->
-      fs.writeFileSync path.resolve('./spec/fixtures/native-cache/file-5'), """
+      fs.writeFileSync path.resolve(__dirname + '/fixtures/native-cache/file-5'), """
       module.exports = function () { return "file-5" }
       """
 
     afterEach ->
-      fs.unlinkSync path.resolve('./spec/fixtures/native-cache/file-5')
+      fs.unlinkSync path.resolve(__dirname + '/fixtures/native-cache/file-5')
```

Comment:
I'm not sure there's actually a good reason to call `path.resolve` here. I think it's meant to be called with multiple arguments.